bootstrap/service.go

```diff
@@ -157,6 +164,27 @@ func (bs bootstrapService) Add(ctx context.Context, token string, cfg Config) (C
 	cfg.State = Inactive
 	cfg.ThingKey = mgThing.Credentials.Secret
 
+	channelsPage, err := bs.sdk.ChannelsByThing(cfg.ThingID, pm, token)
+	if err != nil {
+		return Config{}, errors.Wrap(errCheckChannels, err)
+	}
```

Comment:
This will not work , if thing is connect to 100 of channels. 
By default the query limit is 10. So with this `bs.sdk.ChannelsByThing(cfg.ThingID, pm, token)` we could not get all the 100 channels of thing. 
There are two options: 
- Iterate on `channelsPage.Total` and get all the channels
- New API endpoint things to verify the thing and channel are connected. (This option looks for me most efficent way)